executor/runtime/types/container.go

```diff
@@ -1185,15 +1196,6 @@ func populateContainerEnv(c Container, config config.Config, userEnv map[string]
 			env[metadataserverTypes.NetflixIPv6sEnvVarName] = vpcAllocation.IPV6Address.Address.Address
 		}
 
-		if vpcAllocation.ElasticAddress != nil {
-			env[metadataserverTypes.EC2PublicIPv4EnvVarName] = vpcAllocation.ElasticAddress.Ip
```

Comment:
Is this env being set through some other means?

Reply:
See line 1197